docs/validator/economics.md

```diff
@@ -30,11 +30,11 @@ You can obtain a validator *seat* by:
 
 ### Stake your NEAR tokens
 
-Use [NEAR Shell](../development/near-clitool.md) to append your staking transaction to the current `proposals`. At the end of the `epoch`, if your proposal is sufficient, your node will become a validator - and will begin to generate rewards after one more epoch. If needed, you can increase anytime your `proposal` by signing a larger staking transaction using the same *stakingKey*.
+Use [NEAR Shell](../development/near-clitool.md) to generate your staking transaction and issue your `proposal` to become validator. At the end of the `epoch`, if your proposal is large enough, your node will become a validator - and will begin to generate rewards after one more epoch. If needed, you can increase anytime your `proposal` by signing a larger staking transaction using the same *stakingKey*.
```

Comment:
I suggest you switch "generate" and "issue" in this sentence. It makes more sense that way.